notebooks/transport/demo_data.go

```diff
@@ -0,0 +1,199 @@
+package transport
```

Comment:
This entire file is just for generating some demo data to experiment with the frontend and won't be permanent.